recap-loops/index.md

```diff
@@ -183,7 +183,32 @@ for (int number : numbers) {
 
 
 ## Exercises
-### Exercise 1: Multiplication Table
+
+### Sum of an array.
+
+Write a method that takes an array of integers and returns the sum of the elements of said array.
+
+### Average
+
+The average of a collection of numbers is the sum divided by the number of elements. Write a method that takes an array of
+integers and returns the average. Make sure to reuse the sum method that you have written above.
+
+### Count the `char`
+
+Write a method that takes a `String` and a `char` and returns the number of times you can find that character in that
+string.
+
+### FizzBuzz
+
+Write a program that prints the numbers from 1 to 100. But for multiples of three print “Fizz” instead of the number and for the multiples of five print “Buzz”. For numbers which are multiples of both three and five print “FizzBuzz”.
+Share your code (also with each other) using git.
```

Comment:
```suggestion
```

This was their entrance exam when they applied for the course.

Reply:
Thanks, good to know :)